book/0 - Introduction/1 - Introduction.html

```diff
@@ -9,3 +9,13 @@
 <p>
 	The majority of this manual was written by <a href="https://github.com/bearbin">@bearbin</a> and it was assembled by a generator written by <a href="https://github.com/bearbin">@bearbin</a> as well.
 </p>
+
+<h3>Resources</h3>
+
+<ul>
+	<li><a href="http://cuberite.org">Cuberite homepage</a></li>
+	<li><a href="http://book.cuberite.org">User manual</a></li>
```

Comment:
Is it really necessary to have a link to yourself? (also it should be "User's Manual")